test/github_pull_requests_test.rb

```diff
@@ -90,6 +90,24 @@ def test_pull_request_status_test_success
     assert receive_test({}, github_slug: "pbrisbin/foo")[:ok], "Expected test of pull request to be true"
   end
 
+  def test_pull_request_status_test_success_and_comment_success
+    @stubs.post("/repos/pbrisbin/foo/statuses/#{"0" * 40}") { |_env| [422, {}, ""] }
```

Comment:
thanks, super helpful! there were enough changes to existing tests that I just sort of ported this file over manually from the version on this branch, which led to some scenario description strings being a bit different than you did, but I think it's pretty much otherwise the same